pilot/cmd/pilot-discovery/main.go

```diff
@@ -106,8 +108,12 @@ func init() {
 		"Select a namespace where the controller resides. If not set, uses ${POD_NAMESPACE} environment variable")
 	discoveryCmd.PersistentFlags().StringSliceVar(&serverArgs.Plugins, "plugins", bootstrap.DefaultPlugins,
 		"comma separated list of networking plugins to enable")
+
+	// MCP client flags
 	discoveryCmd.PersistentFlags().StringSliceVar(&serverArgs.MCPServerAddrs, "mcpServerAddrs", []string{},
-		"comma separated list of mesh config protocol server addresses")
+		"comma separated list of MCP server addresses with "+
+			"mcp:// (secure) or mcpi:// (insecure) schema, e.g. mcp://istio-galley.istio-system.svc:9901")
```

Comment:
When will it be insecure?
If we need insecure, shouldn't we use mcp for insecure and mcps for secure?

Reply:
Sounds reasonable.